src/scripts/zone_text.gd

```diff
@@ -1,32 +1,3 @@
 extends Label
 
-func _ready():
```

Comment:
Moved this into the game HUD, I'm not sure why the complexity was added of another layer, when the UI is typically responsible for relaying data to to the player? It also simplifies it because we don't need to call `_on_Sub_depth_status()` in multiple places, just in one.